onnxruntime/core/mlas/lib/amd64/ErfKernelFma3.asm

```diff
@@ -0,0 +1,567 @@
+;
+;--
+
+        .xlist
+INCLUDE mlasi.inc
+        .list
+
+        EXTERN  MlasMaskMoveAvx:NEAR
+        EXTERN  MlasErfConstants:NEAR
+
+;
+; Structure layout for the tanh constants block.
+;
+
+ErfConstants STRUCT
+    ErfUpperAbsRange DWORD ?
+    ErfSplitBoundary DWORD ?
+    ErfSMALL_P0 DWORD ?
+    ErfSMALL_P1 DWORD ?
+    ErfSMALL_P2 DWORD ?
+    ErfSMALL_P3 DWORD ?
+    ErfSMALL_P4 DWORD ?
+    ErfSMALL_P5_Minus_One DWORD ?
+    ErfMaxInt DWORD ?
+    ErfBIG_P0 DWORD ?
+    ErfBIG_P1 DWORD ?
+    ErfBIG_P2 DWORD ?
+    ErfBIG_P3 DWORD ?
+    ErfBIG_P4 DWORD ?
+    ErfBIG_P5 DWORD ?
+    ErfBIG_P6_Minus_One DWORD ?
+    ErfNegZero DWORD ?
+    ErfOne DWORD ?
+
+    Exp_UpperRange DWORD ?
+    Exp_LowerRange DWORD ?
+    Exp_Log2Reciprocal DWORD ?
+    Exp_log2_hi DWORD ?
+    Exp_log2_lo DWORD ?
+    Exp_P0 DWORD ?
+    Exp_P1 DWORD ?
+    Exp_P2 DWORD ?
+    Exp_P3 DWORD ?
+    Exp_P4 DWORD ?
+    Exp_P5 DWORD ?
+    Exp_P6 DWORD ?
+    Exp_C DWORD ?
+    Exp_X7F DWORD ?
+ErfConstants ENDS
+
+;
+; Stack frame layout for the tanh kernel.
+;
+
+ErfKernelFrame STRUCT
+        ErfBuffer0 OWORD 8 DUP(?)
+        ErfBuffer1 OWORD 8 DUP(?)
+        SavedXmm6 OWORD ?
+        SavedXmm7 OWORD ?
+        SavedXmm8 OWORD ?
+        SavedXmm9 OWORD ?
+        SavedXmm10 OWORD ?
+        SavedXmm11 OWORD ?
+        SavedXmm12 OWORD ?
+        SavedXmm13 OWORD ?
+        SavedXmm14 OWORD ?
+        SavedXmm15 OWORD ?
+        Padding0 QWORD ?
+        Padding1 QWORD ?
+        CountN QWORD ?
+        ReturnAddress QWORD ?
+        PreviousP1Home QWORD ?
+        PreviousP2Home QWORD ?
+        PreviousP3Home QWORD ?
+        PreviousP4Home QWORD ?
+
+ErfKernelFrame ENDS
+
+;++
+;
+; Routine Description:
+;
+;   This routine implements a vectorized kernel for the error
+;   function.
+;
+; Arguments:
+;
+;   Input (rcx) - Supplies the input buffer.
+;
+;   Output (rdx) - Supplies the output buffer.
+;
+;   N (r8)  - Supplies the number of elements to process.
+;
+; Return Value:
+;
+;   None.
+;
+;--
+
+        NESTED_ENTRY MlasErfKernelFma3, _TEXT
+
+        alloc_stack (ErfKernelFrame.ReturnAddress)
+
+        save_xmm128_avx xmm6,ErfKernelFrame.SavedXmm6
+        save_xmm128_avx xmm7,ErfKernelFrame.SavedXmm7
+        save_xmm128_avx xmm8,ErfKernelFrame.SavedXmm8
+        save_xmm128_avx xmm9,ErfKernelFrame.SavedXmm9
+        save_xmm128_avx xmm10,ErfKernelFrame.SavedXmm10
+        save_xmm128_avx xmm11,ErfKernelFrame.SavedXmm11
+        save_xmm128_avx xmm12,ErfKernelFrame.SavedXmm12
+        save_xmm128_avx xmm13,ErfKernelFrame.SavedXmm13
+        save_xmm128_avx xmm14,ErfKernelFrame.SavedXmm14
+        save_xmm128_avx xmm15,ErfKernelFrame.SavedXmm15
+
+        END_PROLOGUE
+
+        lea     rax,MlasErfConstants
+
+        sub     r8,8*4
+        jb      LErfProcessRemainingCount
+
+LComputeErf4x8Loop:
+        vbroadcastss ymm15,ErfConstants.ErfNegZero[rax]
+        vmovups      ymm0,YMMWORD PTR [rcx]        ; original input vx0
```

Comment:
The existing asm style is to start operands at column 17 (four tabs in) unless the operator is too long (like vbroadcastss). Can you fix?

Reply:
No problem. I will fix all tab related issuses.
Sorry that I just squash previous change list into one, should be done in advanced.